projects/packages/my-jetpack/_inc/hooks/use-route-blocker/index.js

```diff
@@ -1,13 +1,15 @@
 import { useEffect } from 'react';
-import { useNavigationType, useNavigate } from 'react-router-dom';
+import { useNavigationType, NavigationType } from 'react-router-dom';
+import { MyJetpackRoutes } from '../../constants';
+import useMyJetpackNavigate from '../use-my-jetpack-navigate';
 
 export default () => {
+	const navigateHome = useMyJetpackNavigate( MyJetpackRoutes.Home );
 	const type = useNavigationType();
-	const navigate = useNavigate();
 
 	useEffect( () => {
-		if ( type === 'POP' ) {
-			navigate( '/', { replace: true } );
+		if ( type === NavigationType.Pop ) {
+			navigateHome( { replace: true } );
```

Comment:
I had noticed in the past if I were to switch between interstitials via changing the URL, the back button would go to the previous interstitial. This seems like it would fix that and always go back to My Jetpack which I think I do like better. I am just wondering is there any instance where we wouldn't want "Go Back" to go to My Jetpack home? I don't think so, just want to be sure

Reply:
Ah! So basically this hook was not used anywhere 😅 

I think what you're describing is a good maintenance item - and I'll note it down!